code/rdb/datastripe.q

```diff
@@ -19,12 +24,26 @@
 
     };
 
+// end of day function - no savedown functionality, just wipes tables and updates date partition
+.rdb.datastripeendofday:{[date;processdata]
+    .rdb.tailsortcomplete:0b;
+    // add date+1 to rdbpartition global
+    .rdb.rdbpartition,:: date+1;
+    .lg.o[`rdbpartition;"rdbpartition contains - ","," sv string .rdb.rdbpartition];
+    t:tables[`.] except .rdb.ignorelist;
+    .lg.o[`clear;"Wiping following tables - ","," sv string t];
```

Comment:
Why are the commas needed in these log messages?

Reply:
The first comma in the log messages is performing a join operation, the second is being used to create a comma-separated list using the `sv` keyword

Reply:
Would this look more clear?
```suggestion
    .lg.o[`rdbpartition;"rdbpartition contains - ", "," sv string .rdb.rdbpartition];
    t:tables[`.] except .rdb.ignorelist;
    .lg.o[`clear;"Wiping following tables - ", "," sv string t];
```